api/client/samples/quick_start.py

```diff
@@ -9,9 +9,9 @@
 def main():
     client = api.client.Client(API_HOST, ACCESS_TOKEN)
 
-    selected_entities = { u'region_id': 1038, # Cape Verde
-                          u'item_id': 5187, # Management of donkey manure
-                          u'metric_id': 5590032 } # Total Emissions Quantity (mass)
+    selected_entities = { u'region_id': 1029, # Brazil
```

Comment:
We already have brazil soybeans and brazil sugar examples in the crop models so this might get confusing...  Let's use something else here so it's easy to refer to them